views/show.blade.php

```diff
@@ -84,12 +84,12 @@
 @stop
 
 @section('css')
-{!! Asset::styles('logviewer') !!}
+    {!! HTML::style(asset('styles/logviewer.css') !!}
```

Comment:
Wow, yeah, the use of asset() was an oversight. I thought I had removed all uses of it. I didn't realize the HTML facade wasn't a dependency.